core/pom.xml

```diff
@@ -481,6 +481,14 @@
           <groupId>commons-logging</groupId>
           <artifactId>commons-logging</artifactId>
         </exclusion>
+        <exclusion>
+          <groupId>org.codehaus.jackson</groupId>
+          <artifactId>jackson-mapper-asl</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>org.codehaus.jackson</groupId>
+          <artifactId>jackson-core-asl</artifactId>
+        </exclusion>
```

Comment:
May I ask why this is added to `core` module instead of `sql/hive` module?

Reply:
In order to keep consistent with branch-3.4.

Reply:
Thank you for the reply, @wangyum .